components/feature/awesomebar/src/main/java/mozilla/components/feature/awesomebar/provider/SearchActionProvider.kt

```diff
@@ -25,7 +26,7 @@ class SearchActionProvider(
     private val showDescription: Boolean = true,
     private val searchEngine: SearchEngine? = null
 ) : AwesomeBar.SuggestionProvider {
-    override val id: String = java.util.UUID.randomUUID().toString()
+    override val id: String = UUID.randomUUID().toString()
```

Comment:
Same with this one, I like the change but maybe in a separate PR with the one below?